### DISCUSSION TITLE
Game not saving all the options

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1382] [Question and answer] [answered]
_Hello_,
I have a problem with the saving system. Every time I re-open the server, not all the options are saved (for example the dead bodies and the intel are saved but not the enemy faction). I will wait for an answer, thanks in advance!

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> but not the enemy faction

Enemy faction can't be change after a first save because enemy already spawned are saved in city

Cheers

**Reply by Vsauce07** [reply]
_Hello_,
I said that badly. I mean after I created the server from scratch, I chose the faction, I played for a while, I closed it and after I reopen it, the previously chosen faction is not saved, but the intel are

**Reply by Vdauphin** [reply]
Oh that how Arma works, at each restart, A3 take the default mission parameter
You need to change the default one here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L102

**Reply by Vsauce07** [reply]
Okay, thank you for the answer!
